tests/sls/homu/valid_repos.py

```diff
@@ -0,0 +1,39 @@
+from urllib.error import URLError
+from urllib.request import Request, urlopen
+
+import toml
+
+from tests.util import Failure, Success
+
+
+def repoExists(identifier):
+    '''
+    Consumes a repo identifier string in the form of owner/name and returns a
+    boolean indicating whether the request to check if the repository exists
+    on github was successful (200) or not
+    '''
+    try:
+        endpoint = "https://github.com/{}".format(identifier)
```

Comment:
nit: `endpoint` -> `url`. This can also be inlined into the `Request` constructor if you want.